project-extender/src/providers/jira-cloud-provider/JiraCloudProvider.ts

```diff
@@ -19,6 +23,16 @@ class JiraCloudProvider implements ProviderApi {
     if (JiraCloudProvider.accessToken === undefined)
       JiraCloudProvider.accessToken = await getAccessToken(oauthLoginOptions)
 
+    await fetch("https://api.atlassian.com/oauth/token/accessible-resources", {
+      headers: {
+        Accept: "application/json",
+        Authorization: `Bearer ${JiraCloudProvider.accessToken}`,
+      },
+    }).then(async (response) => {
+      await response.json().then(async (data) => {
+        this.cloudID = data[0].id
```

Comment:
Why is 0 hardcoded? What does that mean?

Reply:
data in this case is an array of objects and i need to access the first one to get the cloudID for our domain in order to correctly build the request URL

Reply:
If it's an array, why are we assuming that the user needs the first cloud ID and not the second?